### PR TITLE
[NUI] Make Notification APIs public and fix CA warnings.

### PR DESCRIPTION
**Pull request by rabbitfor**
Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### API Changes ###

 - ACR: https://code.sec.samsung.net/jira/browse/TCSACR-331

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Notification.cs [hunk 1]
```diff
@@ -283,6 +274,43 @@ public void ForceQuit()
...
+            }
+
+            if (onDismissAnimation != null)
+            {
+                onDismissAnimation.Dispose();    
+            }
+
+            base.Dispose(type);
+        }
+
```
Comment (idkiller) [inline src/Tizen.NUI.Components/Controls/Notification.cs:313]: When calling only one method immediately after null check, it is better to use null conditional operator.

```C#
notificationWindow?.Dispose();
timer?.Dispose();
positionSize?.Dispose();
onPostAnimation?.Dispose();
onDismissAnimation?.Dispose();
```

This improves code readability, reduces code, and actually reduces instructions in IL.

Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/Notification.cs:313]: I tried to use "?" here, but when I built it the Code Analysis created [CA2213](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca2213?view=vs-2019) warnings on it. (I don't know why...)

I wanted to keep no Code Analysis warnings in the Notification class so I had to use `if` for null checking.

Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/Notification.cs:313]: I think my code anaylzer is an old one.

Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/Notification.cs]: Done fixing!

**Pull request merged**